pytorch_lightning/core/optimizer.py

```diff
@@ -249,78 +252,79 @@ def _configure_optimizers(
     return optimizers, lr_schedulers, optimizer_frequencies, monitor
 
 
-def _configure_schedulers(
-    schedulers: list, monitor: Optional[str], is_manual_optimization: bool
-) -> List[Dict[str, Any]]:
-    """Convert each scheduler into dict structure with relevant information."""
+def _configure_schedulers_automatic_opt(schedulers: list, monitor: Optional[str]) -> List[Dict[str, Any]]:
+    """Convert each scheduler into dict structure with relevant information, when using automatic optimization."""
     lr_schedulers = []
     default_config = _get_default_scheduler_config()
-    # TODO: move is_manual_optimization check out of for loop
```

Comment:
this should be a separate PR to ensure this one only does one thing

Reply:
I agree

Reply:
Done